linux_os/guide/system/selinux/enable_selinux_bootloader.rule

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: rhel7,fedora
+prodtype: rhel6,rhel7,fedora
 
 title: 'Ensure SELinux Not Disabled in /etc/default/grub'
```

Comment:
@cipherboy curious what your plans are for integrating Grub? Technically this is not correct as `cce@rhel6: 26956-3` is for configuring `grub.conf` whereas `cce@rhel7: 26961-3` is for configuring `/etc/default/grub`

Reply:
This is indicative of a bigger grub-legacy/grub2 split that is ultimately out of scope for this PR. What should really happen is grub-legacy rules go into one folder, and grub2 rules into another, with different titles and different names. Then each operating system and each version can require the specific rules it needs (e.g., grub-legacy-* rules for RHEL6, grub2-* rules for RHEL7, etc.). 

I can drop the changes from this PR if you'd like.

Reply:
@cipherboy please drop the changes where we need to have grub/grub2 split out.

Reply:
@redhatrises Updated.